### DISCUSSION TITLE
Task - Spring Boot - E-commerce Application - sportyshoes.com

### DISCUSSION THREAD

**Discussion by akash-coded** [akash-coded/spring-framework#74] [Tasks]
Create an e-commerce application `sportyshoes-api` using Spring Boot based on the project specification given [here](https://github.com/akash-coded/spring-framework/blob/main/projects/sportyshoes-ecommerce-api-project-description.png). 

You can refer to discussion #60 for DB-related specifications.

The project involves an e-commerce application with a user portal and an admin portal. Users can sign up, sign in, sign out, and perform activities such as product lookup, purchase, editing profile details, and view their order history. Admins can sign in, sign out, and perform administrative duties such as managing users, products, and orders.

Incorporate Swagger2 in the application and enable API and UI documentation. Add a few details about the application, author, endpoints, and models using Swagger annotations. (Hint - You can refer to [this](https://github.com/akash-coded/spring-framework/tree/main/eclipse-projects/spring-boot/spring-boot-swagger-demo-app/src/main/java/com/spring/boot/swagger/demo) demo project for the Swagger2 setup and configuration.)

You can use [this](https://github.com/akash-coded/spring-framework/discussions/58) template to submit your solution.

### To be Submitted:
- GitHub repo link
- Postman collection link
- Packaged JAR file of your Spring Boot app
- Postman collection export file
- DB dump file
- Screenshots

**Comment by kunalki** [comment]
working on it.

**Reply by akash-coded** [reply]
@kpBuddy Any updates on this?

**Comment by vamsi1234567891** [comment]
working on it

**Reply by akash-coded** [reply]
@vamsi1234567891 Any updates on this?

**Comment by GouthamGunji** [comment]
# Git Link 
GIT Link : https://github.com/GouthamGunji/E-Commerce/

# PostMan Collection Link
PostMan Collection Link : https://www.getpostman.com/collections/20f5ca0331932c0fdafa

# DB Dump
[Dump20220511.zip](https://github.com/akash-coded/spring-framework/files/8668259/Dump20220511.zip)

# Postman collection File 
[E-Commerce.postman_collection.zip](https://github.com/akash-coded/spring-framework/files/8668280/E-Commerce.postman_collection.zip)

# Screenshots

<img width="632" alt="image" src="https://user-images.githubusercontent.com/61229602/167810380-e3786104-4eaf-48df-a13f-9c04b76c251f.png">

**Comment by Svishva** [comment]
##### VisweshwaranM 
## Git Hub Link

[https://github.com/Svishva/sporty-shoes-api](https://github.com/Svishva/sporty-shoes-api)

## Postman Collection Link

[Link](https://go.postman.co/workspace/TVS-Next-Training~3c3e133a-de81-4b4f-9ce6-d73eb8faa2ef/collection/20201567-4f541855-145c-4cb6-b44f-2694f40813e4?action=share&creator=20201567)

## Jar
[Sporty-Shoes-Api.zip](https://github.com/akash-coded/spring-framework/files/8672930/Sporty-Shoes-Api.zip)


## Postman Collection
[sporty-shoes-api.postman_collection.zip](https://github.com/akash-coded/spring-framework/files/8672926/sporty-shoes-api.postman_collection.zip)


## DB dump

[dump-sporty_shoes-202205120049.zip](https://github.com/akash-coded/spring-framework/files/8672935/dump-sporty_shoes-202205120049.zip)

**Comment by Thiripurasundari-1223** [comment]
# Shortyshoes Application
GitHub repo link
Packaged JAR file of your Spring Boot app
````java
https://github.com/Thirpurasundari/Sportyshoes
````
# Postman collection link
````java
 https://www.getpostman.com/collections/fd0d647ad8044aff8086
`````
# DB Dump
[Dump20220511.zip](https://github.com/akash-coded/spring-framework/files/8667267/Dump20220511.zip)

# Postman collection export file
[SportyShoes.postman_collection.zip](https://github.com/akash-coded/spring-framework/files/8667283/SportyShoes.postman_collection.zip)

# Screenshot

<img width="636" alt="Screenshot_18" src="https://user-images.githubusercontent.com/103409420/167848516-5030f411-def5-4669-9d31-41588806db84.png">

**Comment by AgeshAalam** [comment]
# DB Dump
[Dump20220511.zip](https://github.com/akash-coded/spring-framework/files/8669609/Dump20220511.zip)

# Postman collection
[SportyShoes.postman_collection.zip](https://github.com/akash-coded/spring-framework/files/8669614/SportyShoes.postman_collection.zip)

# GitHub
https://github.com/AgeshAalam/sportyshoesapplication

# Postman collections URL
https://www.postman.com/interstellar-comet-647553/workspace/ecommerce/collection/8046227-441644b8-e165-4034-99a3-e49b5b1cdbfe?action=share&creator=8046227

![image](https://user-images.githubusercontent.com/44727659/168051955-2ddc75de-81b1-4c3b-8cc4-1456ba329306.png)
![Swagger](https://user-images.githubusercontent.com/44727659/168052123-bfc987c7-8544-4e17-9cb5-3db06b6857e6.PNG)

**Comment by Anandhi22** [comment]
## GitHub 

GitHub Link : https://github.com/Anandhi22/E-Commerce---Sporty-Shoes

## PostMan Collection Link

PostMan Collection Link : https://www.getpostman.com/collections/2f7110cdb80cff117e7d

## DB Dump 

[Dump20220511.zip](https://github.com/akash-coded/spring-framework/files/8670345/Dump20220511.zip)

## PostMan Collection File

[Spring-Boot-Sporty-Shoes.postman_collection.zip](https://github.com/akash-coded/spring-framework/files/8670363/Spring-Boot-Sporty-Shoes.postman_collection.zip)



# Screenshot

## GetMapping of Search Product by ProductId

![Screenshot (72)](https://user-images.githubusercontent.com/85279205/167862494-4143198e-ae9a-43eb-b59f-122a8527bb86.png)

**Comment by KalaiselviRajavadivel** [comment]
# Git Link

https://github.com/KalaiselviRajavadivel/SportyShoes

# PostMan Collection Link

https://www.getpostman.com/collections/d34605acfe7d1bdc9147

# PostMan Collection File
[SportyShoes.postman_collection.zip](https://github.com/akash-coded/spring-framework/files/8670558/SportyShoes.postman_collection.zip)

# DB Dump
[dumps.zip](https://github.com/akash-coded/spring-framework/files/8670545/dumps.zip)

# Swagger Api Docs
[Swagger-api-docs.txt](https://github.com/akash-coded/spring-framework/files/8670594/Swagger-api-docs.txt)

# Screenshots
<img width="960" alt="Snap1" src="https://user-images.githubusercontent.com/105358230/167868715-902dadfb-86a8-4f9f-bc7f-c156aec5cbd5.PNG">
<img width="942" alt="snap2" src="https://user-images.githubusercontent.com/105358230/167868819-bfbb952a-0aa7-455c-8318-d31d7c286106.PNG">

<img width="671" alt="Capture" src="https://user-images.githubusercontent.com/105358230/167871241-486c6239-bd8f-4c78-9b61-70077804c19b.PNG">

**Comment by Narutoram** [comment]
Git Link
https://github.com/Narutoram/sportyshoes/tree/master/api

PostMan Collection Link

https://www.getpostman.com/collections/5a2fa37de7d075c97939

PostMan Collection File

[Sportyshoes.postman_collection.zip](https://github.com/akash-coded/spring-framework/files/8672226/Sportyshoes.postman_collection.zip)
![sportyshoes](https://user-images.githubusercontent.com/51920058/167912286-73d759bb-4a07-4887-9fe3-83d7a7c5fbd8.png)
![Sportyshoes 2](https://user-images.githubusercontent.com/51920058/167912322-1677445c-56eb-4573-8978-5a6674d5a5
![Swagger](https://user-images.githubusercontent.com/51920058/167989833-4e358dc9-6660-42e4-95ff-f42300d5435b.png)
eb.png)

**Comment by vamsi1234567891** [comment]
### Zip file
[sportyshoes-api.zip](https://github.com/akash-coded/spring-framework/files/8675190/sportyshoes-api.zip)

###Prostman links
[ECommerce.postman_collection.txt](https://github.com/akash-coded/spring-framework/files/8675287/ECommerce.postman_collection.txt)

###Prostman zip
[ECommerce.zip](https://github.com/akash-coded/spring-framework/files/8675317/ECommerce.zip)

##SreenShoot
![postman screen](https://user-images.githubusercontent.com/103091206/168001856-280afb22-6671-421f-8abb-f119e08efa9a.JPG)

![swagger screen](https://user-images.githubusercontent.com/103091206/168002033-34b2bfbb-711f-4786-9b16-70ebcae186c5.JPG)

**Comment by pravallikamutyala** [comment]
![2022-05-17 (41)](https://user-images.githubusercontent.com/48113213/168857890-59b3452d-634e-46b3-87a6-ea9540886a1e.png)
![2022-05-17 (42)](https://user-images.githubusercontent.com/48113213/168857893-638cd047-e973-43f7-a7c6-3f1b449a8654.png)
![2022-05-17 (43)](https://user-images.githubusercontent.com/48113213/168857897-ade0f2a2-70ed-46ce-be58-d9f8f463cc91.png)
![2022-05-17 (44)](https://user-images.githubusercontent.com/48113213/168857899-8a0cf9a4-59c7-46b8-8575-142cfd73662d.png)
![2022-05-17 (45)](https://user-images.githubusercontent.com/48113213/168857903-0d0af203-b661-4c57-8558-003f7fef84cb.png)

**Reply by pravallikamutyala** [reply]
I forgot to add these screenshots in LMS. Thank you, Sir.
